JavaDoc поддержка новых терминов ФФД 1.05 и переход на 20% налоговую ставку

Lytkini opened the pull request.

Lytkini committed:
JavaDoc FFD + VAT

Lytkini committed:
Merge branch 'develop' into JavaDoc_FFD_VAT

notxcain commented on src/main/java/ru/evotor/framework/receipt/TaxNumber.kt:
```diff
@@ -2,11 +2,35 @@ package ru.evotor.framework.receipt
+ */
 enum class TaxNumber(val value: BigDecimal) {
+    /**
+     * Основная ставка 18%. С первого января 2019 года может указывать как на 18%, так и на 20% ставку.
```
А почему не завести `VAT_20`?

Lytkini replied:
@xcam, прокомментируй, пожалуйста.

xcam replied:
Потому что тогда каждому клиенту придется руками у товаров указать новую ставку или каждой ТУ придется перегрузить товары с новыми ставками

Lytkini committed:
Merge branch 'develop' into JavaDoc_FFD_VAT

Lytkini committed:
Merge branch 'develop' into JavaDoc_FFD_VAT

Lytkini committed:
Merge branch 'develop' into JavaDoc_FFD_VAT

ArTemmey committed:
refactored fiscal requisites

ArTemmey committed:
begin model

xcam commented on src/main/java/ru/evotor/framework/payment/PaymentType.java:
```diff
@@ -10,12 +10,12 @@
 
     /**
-     * Наличными
+     * Наличными срдествами
```
опечатка

Lytkini replied:
Поправил.

xcam reviewed (changes requested):
Есть замечание

Lytkini committed:
Typo fix

ArTemmey committed:
added method to KktApi

ArTemmey committed:
added method registerCorrectionReceipt to KktApi

ArTemmey committed:
good exception codes

ArTemmey committed:
merge develop

ArTemmey committed:
renamed service

ArTemmey committed:
jdoc

ArTemmey committed:
some refactor

Lytkini committed:
Чек возврата на основании

Lytkini committed:
Merge branch 'develop' into JavaDoc_FFD_VAT

Lytkini committed:
Dokka task modified, packages description started

ArTemmey committed:
fixed

ArTemmey committed:
empty commit

Commit:
Packages and IntegrationService described. Dokka Settings added. Position fields corrected

ArTemmey committed:
Merge branches 'STDEV-981' and 'develop' of https://github.com/evotor/integration-library into STDEV-981

ArTemmey committed:
STDEV-981 merged develop

Commit:
Merge branch 'develop' into JavaDoc_FFD_VAT

Commit:
Merge branch 'STDEV-981' into JavaDoc_FFD_VAT

Commit:
Merge branch 'develop' into JavaDoc_FFD_VAT

Commit:
A lot of new javadoc

Commit:
Documentation for commands and events

Added documentation for command and events to accopany refactoring of integration points topic

Commit:
Dokka task + some docs

Commit:
Javadoc update

Commit:
Merge branch 'develop' into JavaDoc_FFD_VAT
